README.md

```diff
@@ -8,15 +8,16 @@ Azure Data Studio is a data management tool that enables you to work with SQL Se
 
 ## **Download the latest Azure Data Studio release**
 
-Platform | Link
--- | --
-Windows User Installer | https://go.microsoft.com/fwlink/?linkid=2121609
-Windows System Installer | https://go.microsoft.com/fwlink/?linkid=2121612
-Windows ZIP | https://go.microsoft.com/fwlink/?linkid=2121511
-macOS ZIP | https://go.microsoft.com/fwlink/?linkid=2121611
-Linux TAR.GZ | https://go.microsoft.com/fwlink/?linkid=2121510
-Linux RPM | https://go.microsoft.com/fwlink/?linkid=2121613
-Linux DEB | https://go.microsoft.com/fwlink/?linkid=2121610
+| Platform																|
```

Comment:
Huh - are all these tabs necessary to get it to align correctly?

Reply:
Nope just making it look pretty

Reply:
Looks better on vscode ._.